endpoints/decorated.py

```diff
@@ -65,6 +65,7 @@ def handle_readonly(ex):
 
 
 @app.errorhandler(NotImplementedError)
+@crossorigin()
```

Comment:
Do we need this?

Reply:
Actually I see that the above issue was because I was running the patternfly ui in a dev environment outside the container - removing the decorator since it's not necessary outside of that